### ISSUE TITLE
Loading mesh causes memory leak

### ISSUE THREAD

**Issue by krips89** [qnzhou/PyMesh#19]
I observed that if I load meshes in a loop, it causes memory leak as the memory used by my script seems to increase continuously. This could be due to other factors, but is being caused at least by `load_mesh` function, which I narrowed down. For example, the following code results in the increase in memory with time - which should not happen as, the variable mesh should be destroyed by the end of the loop.

```
filename = './bathtub/train/bathtub_0057.off'
for i in range (10000):
    mesh = pymesh.load_mesh(filename)
```

**Comment by qnzhou** [comment]
Good catch.  It should be fixed now.  Can you pull master and test again?
